lib/components/ListPicker.vue

```diff
@@ -0,0 +1,37 @@
+<template>
+    <select :value="value" ref="listpicker" @change="onSelectedIndexChange($event)" @input="updateValue()">
+        <option v-if="disabledVal" disabled :value="disabledVal">{{ disabledVal }}</option>
+        <option v-for="item in items" :key="item[valueProp]" :value="item[valueProp]">
+                {{ item[labelProp] }}
+        </option>
+    </select>
+</template>
+
+<script>
+
+export default {
+    name: 'ListPicker',
+    props: [
+        'disabledVal', // Works only in web.
```

Comment:
Why do we need this? I can't see it in the api docs.

Reply:
The example in the docs does not list an array of objects. I have added key value props to build just a prototype.

Reply:
I'm talking about disabled here. I think we should remove it.